src/pyodide/internal/python.ts

```diff
@@ -222,9 +75,13 @@ export async function loadPyodide(
   lockfile: PackageLock,
   indexURL: string
 ): Promise<Pyodide> {
-  const emscriptenSettings = getEmscriptenSettings(lockfile, indexURL);
   const Module = await enterJaegerSpan('instantiate_emscripten', () =>
-    instantiateEmscriptenModule(emscriptenSettings)
+    instantiateEmscriptenModule(
```

Comment:
Do you intend to export this call to c++ in a separate PR?

Reply:
Yeah